modules/ROOT/examples/live-demos/full-featured/index.js

```diff
@@ -1,6 +1,5 @@
-
 /* Script to import faker.js for generating random data for demonstration purposes */
-tinymce.ScriptLoader.loadScripts(['https://cdn.jsdelivr.net/npm/faker@5/dist/faker.min.js'], function() {
+tinymce.ScriptLoader.loadScript('https://cdn.jsdelivr.net/npm/faker@5/dist/faker.min.js').then(() => {
```

Comment:
Why was this file updated? This isn't part of the plugins and is going to conflict with other PRs.

- [ ] Revert changes to demos that aren't in the plugins section

Reply:
I have merged in the latest staging changes and used the incoming changes instead of my changes in all of the cases.